pvlib/spectrum/mismatch.py

```diff
@@ -571,3 +571,128 @@ def spectral_factor_caballero(precipitable_water, airmass_absolute, aod500,
     )
     modifier = f_AM + f_AOD + f_PW  # Eq 5
     return modifier
+
+def spectral_factor_pelland(airmass_absolute, clearness_index,
```

Comment:
`clearness_index` needs to be `clearsky_index`. Clearness index is GHI / extraterrestrial; clearsky is GHI / clearsky model GHI.

Strictly speaking, the parameters for this model are tied to the ESRA clear-sky model (Rigollier, Bauer, Wald 2000, in the references of Pelland et al.) But that model looks complicated. For this addition of `spectral_factor_pelland`, a Note about the parameters would be helpful.

Reply:
@cwhanse Glad you picked up on this; an oversight on my part. I have corrected and added in a notes section with further details on the parameters.